src/colecaoAutor.py

```diff
@@ -0,0 +1,14 @@
+from src.autor import *
+class ColecaoDeAutores:
+
+    def __init__(self):
+        self.__lista = []
+
+    def incluir(self,autor: Autor):
+        if autor in self.__lista:
```

Comment:
Neste `in` aqui, ele vai executar o `__eq__` do próprio Autor? Se for, Mandou bem!

Reply:
É isso mesmo, ele compara o autor com cada autor na lista e por sua vez quando um autor é comparado ele chama o __eq__ que eu fiz a sobrecarga.